actionpack/lib/action_dispatch/http/content_security_policy.rb

```diff
@@ -43,10 +51,12 @@ def call(env)
 
         request = ActionDispatch::Request.new env
 
-        if policy = request.content_security_policy
+        if (policy = request.content_security_policy)
```

Comment:
These cosmetic changes are unrelated and non-conventional for the Rails code base.
```suggestion
        if policy = request.content_security_policy
```